### PR TITLE
Improve page 3

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently, the product cards are not designed to comply with e-commerce regulations.

**What is the new behavior (if this is a feature change)?**
- The product cards have been improved and are now compliant to e-commerce regulations.
- The black wolf product has been visually improved.
- The blue lupi side logo no longer joins the footer on mobile devices.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -287,7 +287,7 @@ class App extends React.Component {
   registerQuestionnaireComplete = () => {
     // Facebook Pixel
     ReactPixel.track("CompleteQuestionnaire");
-    
+
```
Comment (schettn) [inline src/App.js:290]: Remove this newline.

**Review by Aichnerc — COMMENTED** [review]
Looks as good as his coffee tastes like.

File: src/components/molecules/Product/index.jsx [hunk 2]
```diff
@@ -95,27 +96,83 @@ class Shop extends React.Component {
...
+
+        if (!isNaN(parts[0])) {
+          return formatter.format(
+            (variant.edges[this.state.variant.key].node.price /
+              parseInt(parts[0])) *
+              1000
+          );
+        } else {
+          if (parts[0].includes("k")) {
+            const kg = parts[0].split("k");
```
Comment (schettn) [inline src/components/molecules/Product/index.jsx:152]: add newline

**Review by schettn — CHANGES_REQUESTED** [review]
Check it out

**Review by schettn — APPROVED** [review]
🚀

**Pull request merged**